src/pages/SetUserGoalPage.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button } from "@material-tailwind/react";
+import { useState, ChangeEvent } from "react";
+import Dropdown from "../components/Dropdown";
+import { Period, periodOptions } from "../models/projectModels";
+import { setUserGoal } from "../storage/userGoalStorage";
+import { useNavigate } from "react-router-dom";
+
+export default function SetUserGoalPage() {
+  let [count, setCount] = useState("");
+  let [period, setPeriod] = useState<Period>("daily");
+
+  const navigate = useNavigate();
+
+  function handleCountChange(event: ChangeEvent<HTMLInputElement>) {
+    const input = (event.target as HTMLInputElement).value;
+    const numbersOnlyInput = input.replace(/\D/g, "");
+    setCount(numbersOnlyInput);
+  }
+
+  function handlePeriodSelect(period: string) {
+    setPeriod(period as Period);
+  }
+
+  async function handleSubmit() {
+    alert("A goal was submitted: " + count);
```

Comment:
Should we log instead of using an alert?

Reply:
Sure, this was mostly copy and pasted from the project creation